RemoteSpiNNaker/RemoteSpiNNakerWeb/src/main/java/uk/ac/manchester/cs/spinnaker/rest/utils/RestClientUtils.java

```diff
@@ -88,9 +90,24 @@ private RestClientUtils() {
      * @return the client
      */
     protected static ResteasyClient createRestClient(final URL url) {
+
         try {
             SSLContextBuilder builder = new SSLContextBuilder();
-            builder.loadTrustMaterial(null, new TrustSelfSignedStrategy());
+            builder.loadTrustMaterial(new TrustSelfSignedStrategy());
+            String trustStore = System.getProperty(
+                    "remotespinnaker.keystore", null);
+            if (trustStore != null) {
+                String password = System.getProperty(
+                        "remotespinnaker.keystore.password", "");
+                try {
+                    builder.loadTrustMaterial(new File(trustStore),
+                            password.toCharArray());
+                } catch (IOException | CertificateException e) {
+                    log.error("Error loading certificates", e);
+                    throw new RuntimeException(
+                            "Unexpected error loading certificates", e);
```

Comment:
Does this exception get logged? If it does, remove the write to the log just above it. We don't want to log exceptions for a single failure multiple times; things are (more than!) verbose as it is.

Reply:
Yes, probably; I think I coped this from somewhere else...

Reply:
Fixed